lang/en.lyaml

```diff
@@ -1517,6 +1517,10 @@ en:
         inactiveSecondary: "Run {{ command }} to remove inactive accounts from your CLI config"
         unableToDetermine: "Unable to determine if the portal is active"
         pak:
+          incomplete:
+            one: "Personal access key is valid. {{ count }} scope available, but not included in your key."
+            other: "Personal access key is valid. {{ count }} scopes available, but not included in your key."
+          incompleteSecondary: "To add the available scopes, run {{ command}} and re-authenticate your account with a new key that has those scopes. Visit hubspot.com to view selected and available scopes for your personal access key. {{ link }}"
```

Comment:
We refer to it as " visit HubSpot" instead of "visit hubspot.com" (small change but worth the consistency)

Reply:
Understood (I thought the hubspot.com was a little wacky too). The existing copy for when you need to paste in your key is `"Open hubspot.com to copy your personal access key?"` - should we change that also? My thinking is yes, and there are a few others as well. I followed convention with mine 🤷‍♂️

Reply:
Good call, I didn't realize we had those inconsistencies tbh. But yeah it would be great to update that too if you don't mind! (not a blocker though)